### PR TITLE
Header molecule with 3 sections and examples with basic header and ivory header

### PR DESCRIPTION
**Pull request by msal96**

### CODE REVIEW

File: src/ui/molecules/Header/Header.examples.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#### Basic Header
+```js
+const React = require('react')
+const textCmp = text => <p>{text}</p>
```
Comment (mihai-ro) [inline src/ui/molecules/Header/Header.examples.md]: If textCmp is intended to be component, please name it with a capital letter.

Comment (mihai-ro) [reply src/ui/molecules/Header/Header.examples.md]: Also. You could use <Text /> component, that is already available.

File: src/ui/molecules/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  maxWidth,
+  minHeight,
+  minWidth,
+  position,
+  space
+} from 'styled-system'
+
+const Header = ({
+  left,
+  middle,
```
Comment (mihai-ro) [inline src/ui/molecules/Header/Header.js]: Please pass `className` prop as well.

File: src/ui/molecules/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  right,
+  ...rest
+}) => (
+  <StyledHeader {...rest}>
+    { left ?
```
Comment (mihai-ro) [inline src/ui/molecules/Header/Header.js]: Please add the conditinoal inside the sections, as you pass content only.

File: src/ui/molecules/Header/Header.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  ${minWidth}
+  ${position}
+  ${space}
+`
+
+const LeftSection = styled.div`
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-content: center;
```
Comment (mihai-ro) [inline src/ui/molecules/Header/Header.js]: Careful with this rule. You might not want to center the content.

File: src/ui/molecules/Header/Header.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  flex-wrap: wrap;
+`
+const MiddleSection = styled.div`
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-self: center;
```
Comment (mihai-ro) [inline src/ui/molecules/Header/Header.js]: Same here. be careful with the content alignment inside containers. you might now want to center them. 
Same rule applies to <Right /> section

Comment (msal96) [reply src/ui/molecules/Header/Header.js]: You're right, I left only display: flex and flex: 1;
When you want to align your component which is in one from sections, then simply pass align-self.

File: src/ui/molecules/Header/Header.examples.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
+#### Basic Header
+```js
+const React = require('react')
+const { default: styled } = require('styled-components')
+
+const Text = styled.p`
```
Comment (mihai-ro) [inline src/ui/molecules/Header/Header.examples.md]: Please use already existing `<Text />` component.

File: src/ui/molecules/Header/Header.examples.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+  flex: 1;
+  text-align: center;
+`
+
+const HeaderExample = () => {
```
Comment (mihai-ro) [inline src/ui/molecules/Header/Header.examples.md]: Please use implicit return if you have no other code executed before your return

File: src/ui/molecules/Header/Header.examples.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+  />
+}
+  <HeaderExample />
+```
+
+#### Ivory Header
+```js
+const React = require('react')
+const { default: styled } = require('styled-components')
+const { default: logoAnimation } = require('assets/logoAnimation.css')
```
Comment (mihai-ro) [inline src/ui/molecules/Header/Header.examples.md]: Please, don't import css files in examples; it is expected for the component to be animated if the component has an animation rule.

**Pull request merged**